src/common/AbstractObject.cpp

```diff
@@ -323,12 +324,12 @@ void AbstractObject::setEditor(const std::string & editor)
 	if (!editor.empty()) {
 		if (gsoapProxy2_0_1 != nullptr) {
 			if (gsoapProxy2_0_1->Citation->Editor == nullptr)
-				gsoapProxy2_0_1->Citation->Editor = soap_new_std__string(gsoapProxy2_0_1->soap, 1);
+				gsoapProxy2_0_1->Citation->Editor = gsoap_resqml2_0_1::soap_new_std__string(gsoapProxy2_0_1->soap, 1);
 			gsoapProxy2_0_1->Citation->Editor->assign(editor);
 		}
 		else {
```

Comment:
Longer term, there will more than two gsoap proxies. An if else would not be sufficient.
I know it is not a change of this PR : that's only for information during (global and not only diff) review.

Reply:
Ok.